components/layout/display_list/builder.rs

```diff
@@ -1464,7 +1464,7 @@ impl Fragment {
             let background_color = style.resolve_color(style.get_background().background_color);
             let base = state.create_base_display_item(
                 stacking_relative_border_box,
-                clip,
+                stacking_relative_border_box,
```

Comment:
Should the title of this commit be "WebRender rectangle items are *not* sized based on their clipping rectangles"? I'm not sure what the commit message means here, if you're removing `clip`.

Reply:
The second argument is the clipping rectangle, which controls the size of the rectangle.

Reply:
OK, I guess that's fine.